README.md

```diff
@@ -1,111 +1,146 @@
-# IBM CIS Terraform Module
+# IBM Cloud Internet Services (CIS) Module
 
-Collection of modules that make it easier to provision Cloud Internet Services and its resources on IBM Cloud Platform:
+This module provisions an IBM Cloud Internet Services (CIS) instance and allows to add the following into the CIS instance.
```

Comment:
It is not fully accurate as the module itself does not allow adding capabilities, rather that is submodules in this repository. I think this should be named indicating that this repo contains a collection of modules.

Reply:
Indicated about submodules.